feincms_extensions/render_json.py

```diff
@@ -0,0 +1,18 @@
+from feincms import extensions
+
+
+class Extension(extensions.Extension):
+    def handle_model(self):
+        cls = self.model
+
+        def render_json(self, request):
+            """Render the feincms regions into a dictionary."""
+
+            def region_data(region):
+                content_list = getattr(self.content, region.key)
+                return [content.json(request=request) for content in content_list]
```

Comment:
Should we be handling the case where a content type does not have a `json` method? or perhaps a check could be added to the model to make sure that this is not possible?